web-server/src/main/kotlin/io/redgreen/tumbleweed/web/observablehq/BilevelEdgeBundlingGraph.kt

```diff
@@ -11,6 +12,15 @@ data class BilevelEdgeBundlingGraph(
       return jacksonObjectMapper()
         .readValue(json, BilevelEdgeBundlingGraph::class.java)
     }
+
+    fun isValidJson(json: String): Boolean {
```

Comment:
Possible to add a test for this? If I understand correctly, this checks if the given text is JSON and does not validate if the JSON document can be converted into a `BilevelEdgeBundlingGraph`?

Reply:
We are trying to do this, because `twd` cannot visualise the JSON unless it is of a specific format.

Reply:
Yeah, I was debating about that. Making that change.

Reply:
I went with a simple test to verify that function. Let me know if you have something else in mind.